content-security-policy/embedded-enforcement/embedding_csp-header.https.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
This file is of type `https.html` which I copied from the upgrade-insecure-request, but not sure it is necessary here?